flask_HW6/products.py

```diff
@@ -0,0 +1,59 @@
+from fastapi import Request, APIRouter
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+import pandas as pd
+from pydantic import BaseModel, Field
+from store_models import database, products
+
+router = APIRouter(prefix="/products", tags=["products"])
+templates = Jinja2Templates(directory='templates')
+
+
+class ProductIn(BaseModel):
+    title: str = Field(max_length=80)
+    description: str = Field(max_length=512)
+    price: float = Field(gt=0, le=999999)
+
+
+class Product(BaseModel):
+    id: int
+    title: str = Field(max_length=80)
+    description: str = Field(max_length=512)
+    price: float = Field(gt=0, le=999999)
```

Comment:
Модели, кстати, мало чем отличимы, следовательно, одна может наследоваться от другой.
```suggestion
class Product(ProductIn):
    id: int
```